### ISSUE TITLE
Doesn't detect use of 'latest' or missing tag

### ISSUE THREAD

**Issue by roadSurfer** [replicatedhq/dockerfilelint#113]
The following `FROM` definitions should all trigger a message:
```
FROM some.registry:1234/repo/image
FROM some.registry:1234/repo/image as example

FROM repo/image:latest as example
FROM some.registry/repo/image:latest as example
FROM some.registry:1234/repo/image:latest
FROM some.registry:1234/repo/image:latest as example
```
The presence of the port number for the registry and the use of `as` seems to cause the parser to not catch the missing tag or use of `latest`.
